backend/src/db/migrations/libs/constants/genres.ts

```diff
@@ -0,0 +1,24 @@
+const GENRES = [
+  { name: 'Science Fiction', key: 'sci-fi' },
```

Comment:
Use snake_case for keys, change it for all keys with kebab-case
```suggestion
  { name: 'Science Fiction', key: 'sci_fi' },
```